povm_toolbox/sampler/povm_sampler_job.py

```diff
@@ -88,6 +90,84 @@ def result(self) -> PrimitiveResult[POVMPubResult]:
 
         return res
 
+    def save_metadata(self, filename: str | None = None) -> None:
+        """Save the metadata of the :class:`.POVMSamplerJob` instance into a JSON file.
+
+        Args:
+            filename: name of the file where to store the metadata.
+        """
+        if filename is None:
+            filename = f"job_metadata_{self.base_job.job_id()}.pkl"
+        with open(filename, "wb") as file:
+            pickle.dump(
+                {
+                    "base_job_id": self.base_job.job_id(),
+                    "metadata": self.metadata,
+                },
+                file,
+            )
+        print(f"Job metadata successfully saved in the '{filename}' file.")
+
+    @staticmethod
+    def load_metadata(filename: str) -> tuple[str, list[POVMMetadata]]:
+        """Load the metadata of a :class:`.POVMSamplerJob` instance from a JSON file.
+
+        Args:
+            filename: name of the file where the metadata is stored.
+
+        Returns:
+            The ID of the internal :class:`.qiskit.primitives.BasePrimitiveJob` object
+            and the list of :class:`.POVMMetadata` objects associated to the originally
+            submitted pubs.
+        """
+        with open(filename, "rb") as file:
+            data = pickle.load(file)
+        return tuple(
+            [
+                data["base_job_id"],
+                data["metadata"],
+            ]
+        )
```

Comment:
```suggestion
        return (
            data["base_job_id"],
            data["metadata"]
        )
```
No need to make this a list first and then convert to a tuple